### PR TITLE
III-4806 Update guzzle

### PR DESCRIPTION
**Pull request by bertramakers**
### Changed

- Updated `guzzlehttp/guzzle` from `v6.3` to `v7.4`
- Updated `guzzlehttp/promises` from `v1.4.0` to `v1.5.1`
- Updated `guzzlehttp/psr7` from `v1.4` to `v2.4`
- Updated `http-interop/http-factory-guzzle` from `v1.0.0` to `v1.2.0`
- Updated `php-http/httplug` from `v1.1.0` to `v2.3.0`
- Updated `php-http/client-common` from `v1.10.0` to `v2.5.0`
- Replaced `php-http/guzzle6-adapter` with `php-http/guzzle7-adapter`

- Updated `auth0/auth0-php` from `v5.7.0` to `v8.2.1`
- Replaced `sentry/sdk` `v2.5.2` with `sentry/sentry` `v3.6.0`
- Updated `aws/aws-sdk-php` from `v3.194.2` to `v3.226.0`
- Updated `willdurand/geocoder` from `v3.3.2` to `v4.5.0`

### Added

- `geocoder-php/google-maps-provider` (was extracted from `willdurand/geocoder` in a major version update)

### Removed

- `egeloen/http-adapter`
- Unused `http.guzzle` service (and its `HttpServiceProvider` class)

---
Ticket: https://jira.uitdatabank.be/browse/III-4806

### CODE REVIEW

**Comment by bertramakers** [comment]
The acceptance tests are all reporting `500` responses when I run them, so I will need to look into that before we can merge this. Changing this to a draft for now.

File: app/Http/HttpServiceProvider.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
 namespace CultuurNet\UDB3\Silex\Http;
 
 use GuzzleHttp\Client as GuzzleClient;
-use Http\Adapter\Guzzle6\Client as Guzzle6ClientAdapter;
+use Http\Adapter\Guzzle7\Client as Guzzle6ClientAdapter;
```
[the remark this answers is not in the document]
Comment (bertramakers) [reply app/Http/HttpServiceProvider.php:8]: Oh yeah I overlooked that alias

[the remark this answers is not in the document]
Comment (bertramakers) [reply app/Http/HttpServiceProvider.php:8]: I'm under the impression that this `guzzle.http` is actually not used anywhere though 🤔

[the remark this answers is not in the document]
Comment (bertramakers) [reply app/Http/HttpServiceProvider.php]: I've removed the service, unless you or @LucWollants can spot a usage. I haven't found one. (Running the acceptance tests to be sure)

**Comment by bertramakers** [comment]
All acceptance tests (still) green after removing the unused guzzle service:

```
339 scenarios (339 passed)
4442 steps (4442 passed)
15m38.947s
```

**Review by LucWollants — APPROVED** [review]
Looks good!

File: app/Http/HttpServiceProvider.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (LucWollants) [reply app/Http/HttpServiceProvider.php]: I don't see any usages.

**Pull request merged**